lib/rules/sizes/index.js

```diff
@@ -0,0 +1,67 @@
+const { isArray } = require("lodash");
+const { parse } = require("postcss-values-parser");
+const {
+  createPlugin,
+  utils: { report, ruleMessages, validateOptions }
+} = require("stylelint");
+
+const {
+  absoluteLengths,
+  fontRelativeLengths,
+  viewportRelativeLengths
+} = require("../../reference/valueSets");
+
+const lengthUnits = [
+  ...absoluteLengths,
+  ...fontRelativeLengths,
+  ...viewportRelativeLengths
+];
+const ruleName = "scales/sizes";
+const messages = ruleMessages(ruleName, {
+  expected: (value, scale) => `Expected "${value}" to be one of "${scale}"`
+});
+
+// Properties to apply the scale to
+const propertyFilter = /(width|height)/;
```

Comment:
Should this be this? Forcing the properties to END with?

```suggestion
const propertyFilter = /(width$|height$)/;
```